components/dropdown/dropdown-content-mixin.js

```diff
@@ -528,6 +541,13 @@ export const DropdownContentMixin = superclass => class extends LocalizeCoreElem
 		this._hasHeader = e.target.assignedNodes().length !== 0;
 	}
 
+	__onAncestorMutation(mutations) {
+		const opener = this.__getOpener();
+		// ignore mutations that are within this dropdown
+		const reposition = !!mutations.find(mutation => !isComposedAncestor(opener, mutation.target));
```

Comment:
Potentially a silly question - is it possible to not observe these cases initially rather than having to filter them here?

Reply:
I was thinking about that a bit... 

We're using the `{ attributes: true, childList: true, subtree: true }` config so that we'll be notified of any changes in that DOM scope, including the addition/removal of siblings (and/or their sub-tree), or attribute changes that might cause their styles to change. 

We might be able to avoid the filtering here if we use `subtree: false`, but then we'd have to walk this scope's tree and wire-up the mutation observer to each node, excluding the ones in the dropdown's subtree. I think there would also be the added complexity that if any of those nodes were removed, we'd want to stop observing them.

I think it becomes a trade off between a bit of DOM walking only when a mutation occurs (for the single observer per scope) vs DOM walking any time the dropdown opens + observing a lot more nodes.